src/scripts/sync_subclassof.py

```diff
@@ -0,0 +1,38 @@
+"""Create a robot template for purpose of syncing subclassof relations.
```

Comment:
## Command description
Does this sound correct?

Reply:
Hmm, maybe this is what you meant prior when you said "stand alone scripts". I was trying to get clarification on that last meeting.

However, this conflicts a little with what you said last time, and I'm also not sure I agree with the strategy. To my understanding, `mondo-utils` should be for "utils" -- which means something a general function for will be re-used in multiple places. However, other than what's in `scripts/utils.py` in `mondo-ingest`, all of the other Python scripts serve a very specific, single purpose. I do agree with the strategy of DRYing those up though.

Perhaps another idea that you have in mind is that for some kind of cleanliness purpose you'd like `mondo-ingest` to be Python free. But that will mean very frequent updates of `mondo-utils` to PyPi and minor commits in `mondo-ingest` just to bump the version number. This will get more annoying when small bugs are found and we have to bump / re-release / re import multiple times.

Perhaps you are maybe suggesting that I use less Python and more of other tools, but that is in conflict with with our directive to use OAK. It is often best used via the Python CLI, and I am heavily relying on it here (instead of SPARQL).

Additionally, you mentioned that we will continue to have "stand alone" scripts in that, what we are looking for is a `mondo-ingest` (not `mondo-utils`) `click` CLI. However, our Python code will still be modular. So the click command for "sync" or "sync subclass" can still be in a file of its own.